frontend/talentsearch/src/js/types/react-table-config.d.ts

```diff
@@ -121,6 +121,7 @@ declare module "react-table" {
 
   export interface Cell<
     D extends Record<string, unknown> = Record<string, unknown>,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Could you make this a "whole file" exception at the top to help reflect that we don't modify this file?

Reply:
Ok. I will do that!